.changeset/silly-oranges-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"codemirror-json-schema": patch
```

Comment:
I think we should make this a `minor` release just in case anyone is using the custom rendering option for hover and validation, _technically_ an API breakage (and a welcome one). For example, when I switched to full html rendering of the validation errors and hover from the non-rendered markdown, I made it a minor release. It's not a huge deal with pre-release, but this just gives a nod that there's some nice new features in store as well!

```suggestion
"codemirror-json-schema": patch
```